src/components/SendTransactionFeedbackModal.js

```diff
@@ -183,6 +187,7 @@ SendTransactionFeedbackModal.propTypes = {
   onDismissSuccess: PropTypes.func,
   // optional method to be executed when the error modal is dismissed
   onDismissError: PropTypes.func,
+  hide: PropTypes.bool,
```

Comment:
Missing parameter description

Should we add a default value of `false` to it?

Reply:
We should.

Done in cde2e017b02cd1042e5dfa3165d44dd47ef76925

Reply:
Still missing the parameter description

Reply:
Done in 84ca74dacfadedc7450430cae26ca19bfe300a69